### ISSUE TITLE
Mouse scroll wheel not working

### ISSUE THREAD

**Issue by antoinetran** [rdesktop/rdesktop#10]
Hi,

I am using Red Hat 6.5 with rdesktop 1.8.2 (tested with 1.8.3 too) and xrdp 0.5.0-0.13.el6 (tested with 1.6.1-3.el6 too). I tried to connect to the server using rdesktop but the scroll wheel does not work. xev does not show any reactions when scrolling up or down. But using middle button works. What do I have to do for the server to receive wheel events?

Here are some useful details:
a) Windows native rdesktop client -> linux VM xrdp 0.5.0 scroll OK
a) Linux rdesktop 1.8.2 -> Windows VM scroll OK

Thank you.

**Comment by hean01-cendio** [comment]
I took a look into this issue. RDP have two ways of sending keybord/mouse events named as slow-path and fast-path. As I understand fast-path was added to the protocol to take away some overhead in these input event packets. Both ways are supported.

rdesktop only imlpements slow-path and therefor I think that you found a bug in xrdp slow-path handling due to that rdesktop works good with a window terminal server using slow-path sending of key/mouse events.
